react-context/my-app/src/components/useCart.ts

```diff
@@ -0,0 +1,11 @@
+import { useContext } from 'react';
+import { CartContext } from './CartContext';
+
+export function UseCart() {
```

Comment:
Since this is a hook function, it must start with lowercase. It should be named `useCart`.